src/repositories/database.ts

```diff
@@ -13,18 +14,29 @@ export class FirebaseDatabaseRepository implements DatabaseRepository {
 
     async getAll<T>(): Promise<T[]> {
         const docsRef = query(collection(this.firestore, this.collection));
-        const docsSnap = await getDocs(docsRef)
+        return await this.parseQueryResult(docsRef)
+    }
+
+    async create(data: any){
+        await setDoc(doc(collection(this.firestore, this.collection)), data)
+    }
+
+    async findBy<T>(field: string, value: string) :Promise<T[]> {
+        const citiesRef = collection(this.firestore, this.collection);
```

Comment:
hey, I think you copied the name from the firebase example "citiesRef"

Reply:
Ops 😅 Just fixed it